app/models/subject.rb

```diff
@@ -22,6 +22,10 @@ class Subject < ApplicationRecord
   belongs_to :subject_area, foreign_key: :type, inverse_of: :subjects
   has_one :financial_incentive
 
+  scope :modern_languages, ->() do
+    where(type: "ModernLanguagesSubject")
```

Comment:
Sorry. Just realised this is an STI model. Can we do away with this scope altogether?